packages/x-date-pickers/src/internals/components/PickersModalDialog.tsx

```diff
@@ -87,6 +88,19 @@ export function PickersModalDialog(props: React.PropsWithChildren<PickersModalDi
   } = props;
 
   const ActionBar = components?.ActionBar ?? PickersActionBar;
+  const actionBarProps = useSlotProps({
```

Comment:
I am also doing the change on the legacy pickers because it was more complicated to support the typing different than actually developing the feature :laughing: